### ISSUE TITLE
Final project check

### ISSUE THREAD

**Issue by JulianPetersIsCoding** [course-dprep/ratings_series_imdb#45]
This issue is written to be used as an agenda for the meeting on 8/10.

As mentioned in the call the week before, it would be wise to all check the current repository and the grading requirements.
Therefore each team member needs to provide feedback on our current repository as if you grade it. So that we can find optimalizations.

Kind regards,
Julian Peters

**Comment by JulianPetersIsCoding** [comment]
1.Github Repository
1.1 Research motivation
Maybe not the right question, because we ask how but shouldn't it be without the how? We only check if it influences it and not why it happens. Way of deployment is wrong, we knit an html file. Hence this should be changed under "Way of deployment". Furthermore shouldn't only the data_deployment.Rmd contain the research questions and the README only the goal and instructions on how to work with our github?

1.2 Repository structure and documentation
Well done, but simular README and data_deployment.Rmd question.

1.3 Breadth of contributions and way-of-working
Followed correctly.

2.Data preparation & analysis
2.1 Data exploration
Could add a rmarkdown file to the src subfiles analysis, data-preparation and paper explaining what happens in brief? We can extract the information given about those files from the README to structure our work further. And data exploration code can be put in such a rmarkdown file.

2.2 Data preparation
This is done correctly

2.3 Analysis and deployment
Current analysis file lacks dept. Could have coded the analysis so that the regression and plot are saved as a picture and those are called into the final paper in data_deployment.Rmd. This should be done by a function.

3. Source code and automation
3.1 Source code quality
src/data-preparation all files are good
src/analysis see comment above
src/data_deployment.Rmd seeing this file twice, hence needs to be deleted one. The most advanced file is good. Except things like title test 1 make is still in it. When you alter a file look further then only adding your part. 

3.2 Degree of automation
The file runs automatic.

**Comment by irisverzijl** [comment]
@JulianPetersIsCoding  about 1.1: with "how" we mean "how much": maybe rephrase the RQ to: "what is the effect of the length of a TV show on its rating?" to make it more clear?
furthermore, according to Tilburg Science Hub the readme should include the motivation and RQ -  [check this link](https://tilburgsciencehub.com/topics/collaborate-share/share-your-work/content-creation/readme-best-practices/)

**Comment by JulianPetersIsCoding** [comment]
@irisverzijl great then I know why that is added! Was unclear for me.

**Comment by irisverzijl** [comment]
Additional remarks:
there is a data_deployment.html (output file) in the ratings_series_imdb\SRC\paper folder while it should be saved in the ratings_series_imdb\SRC\paper folder

**Comment by jibbebeer** [comment]
1.1: Yes agreed with @JulianPetersIsCoding the readme has too much info probably?
1.2 good, except for the double data_deployment.Rmd file, which will be tackled
1.3 good

2.1 proficient
2.2 looks very good
2.3 great update iris!

3.1 I think this is quite well, given all the usage of custom functions, etc. the source code is of good quality
3.2 When the Rmd bug is fixed it is automated and therefore good

**Comment by irisverzijl** [comment]
@jibbebeer  see previous comment about the readme - pointing to tilburg science hub
lets discuss in class
